packages/wonder-blocks-data/src/components/intercept-data.js

```diff
@@ -26,39 +26,43 @@ type BaseProps<TOptions, TData> = {|
     children: React.Node,
 |};
 
-type FulfillRequestProps<TOptions, TData> = {|
-    /**
-     * Called to fulfill a request.
-     * If this returns null, the request will be fulfilled by the
-     * handler of the original request being intercepted.
-     */
-    fulfillRequest: InterceptFulfillRequestFn<TOptions, TData>,
-|};
+// type FulfillRequestProps<TOptions, TData> = {|
+//     /**
+//      * Called to fulfill a request.
+//      * If this returns null, the request will be fulfilled by the
+//      * handler of the original request being intercepted.
+//      */
+//     fulfillRequest: InterceptFulfillRequestFn<TOptions, TData>,
+// |};
 
-type ShouldRefreshCacheProps<TOptions, TData> = {|
-    /**
-     * Called to determine if the cache should be refreshed.
-     * If this returns null, the handler being intercepted will be asked if
-     * the cache should be refreshed.
-     */
-    shouldRefreshCache: InterceptShouldRefreshCacheFn<TOptions, TData>,
-|};
+// type ShouldRefreshCacheProps<TOptions, TData> = {|
+//     /**
+//      * Called to determine if the cache should be refreshed.
+//      * If this returns null, the handler being intercepted will be asked if
+//      * the cache should be refreshed.
+//      */
+//     shouldRefreshCache: InterceptShouldRefreshCacheFn<TOptions, TData>,
+// |};
 
-type Props<TOptions, TData> =
```

Comment:
I think this original formulation would've worked in TS.  I changed this to define a separate `ConditionalProps` type to match the rest of the changes in this PR.